google-cloud-spanner/src/main/java/com/google/cloud/spanner/CompositeTracer.java

```diff
@@ -109,7 +108,7 @@ public void attemptCancelled() {
   }
 
   @Override
-  public void attemptFailed(Throwable error, Duration delay) {
+  public void attemptFailed(Throwable error, org.threeten.bp.Duration delay) {
```

Comment:
nit: Can we import `java.time.Duration` so that we can remove `java.time` in `attemptFailedDuration` method?

Reply:
Done